packages/bpk-component-sliding-drawer/stories.scss

```diff
@@ -0,0 +1,5 @@
+@import '~bpk-mixins';
```

Comment:
Please add the license header to this file

Reply:
fixed in #286